ground/src/main/res/values/strings.xml

```diff
@@ -128,6 +129,7 @@
   <string name="initializing">Initializing…</string>
   <string name="draw_area_task_instruction">Move the map and tap “Add point” to add points around the desired area.</string>
   <string name="close">Close</string>
+  <string name="data_consent_dialog_title">Data collection agreement</string>
```

Comment:
Follow-up questions and suggestions for @amegantz here:
https://github.com/google/ground-android/issues/2497#issuecomment-2286405809